### PR TITLE
Tutorials in PHP

### PR DESCRIPTION
**Pull request by bartlomiejwereszczynski**

### CODE REVIEW

File: content/tutorials/token-authentication.textile [hunk 1]
```diff
@@ -207,11 +243,44 @@ blang[ruby].
...
 h2. Step 5 - Setting up a client to use token authentication
 
 Now that we have a web server running and an authentication endpoint, we are ready to setup the Ably Realtime client that uses token auth.
 
-First we need to set up a simple HTML page that will be served by the web server. Create the following HTML file named @index.html@ in the same folder as @server.js@:
+blang[nodejs].
+  First we need to set up a simple HTML page that will be served by the web server. Create the following HTML file named @index.html@ in the same folder as @server.js@:
+blang[ruby].
+  First we need to set up a simple HTML page that will be served by the web server. Create the following HTML file named @index.html@ in the same folder as @server.rb@:
+blang[php].
```
Comment (mattheworiordan) [inline content/tutorials/token-authentication.textile]: Same comment as the other PR. Are you running this locally and checking that the page renders as expected.  This is necessary please.

File: content/tutorials/token-authentication.textile [hunk 2]
```diff
@@ -224,6 +293,7 @@ First we need to set up a simple HTML page that will be served by the web server
 </html>
 ```
 
+
```
Comment (mattheworiordan) [inline content/tutorials/token-authentication.textile]: What's this for?

File: content/tutorials/token-authentication.textile [hunk 3]
```diff
@@ -346,7 +457,38 @@ blang[ruby].
...
+  ```[php]
+    <?php
+    setcookie('username', null, -1, '/');
+    header('Location: /', true, 302);
+  ```
+
+
+blang[nodejs,ruby].
+  Both the logged in and anonymous HTML panels will currently show on the HTML page, let's add quickly hide or show them based on the cookie set above. Add the following just before the final @</script>@ tag in @index.html@
+blang[php].
```
Comment (mattheworiordan) [inline content/tutorials/token-authentication.textile]: White space issue here again I believe.  Please confirm the output is as expected

File: content/tutorials/history.textile [hunk 4]
```diff
@@ -370,7 +370,11 @@ blang[php].
 
   bc[sh]. git checkout history-php
 
-  And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @history.php@ and running this script with @php history.php@.
+  And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @history.php@, install composer dependencies with:
```
Comment (mattheworiordan) [inline content/tutorials/history.textile]: Change to "install the composer dependencies" please (added the), same below.

File: content/tutorials/history.textile [hunk 5]
```diff
@@ -370,7 +370,11 @@ blang[php].
...
+
+  bc[sh]. composer install
+
+  and running this script with @php history.php@.
```
Comment (mattheworiordan) [inline content/tutorials/history.textile]: and run this script...

File: content/tutorials/token-authentication.textile (hunk 1 above)
Comment (bartlomiejwereszczynski) [inline content/tutorials/token-authentication.textile]: it renders fine, it's indentation that's taken into account

File: content/tutorials/history.textile (hunk 4 above)
Comment (bartlomiejwereszczynski) [inline content/tutorials/history.textile]: changed

File: content/tutorials/history.textile (hunk 5 above)
Comment (bartlomiejwereszczynski) [inline content/tutorials/history.textile]: changed

File: content/tutorials/token-authentication.textile (hunk 2 above)
Comment (bartlomiejwereszczynski) [inline content/tutorials/token-authentication.textile]: removed

File: content/tutorials/token-authentication.textile (hunk 1 above)
Comment (mattheworiordan) [inline content/tutorials/token-authentication.textile]: Ok, good to know, thanks

**Pull request merged**